Applications/DataExplorer/DataView/NetCdfConfigureDialog.cpp

```diff
@@ -352,13 +354,14 @@ void NetCdfConfigureDialog::createDataObject()
 
 	double origin_x = (originLon < lastLon) ? originLon : lastLon;
 	double origin_y = (originLat < lastLat) ? originLat : lastLat;
-	double originNetCdf[3] = {origin_x, origin_y, 0};
+	MathLib::Point3d origin(std::array<double,3>{{origin_x, origin_y, 0}});
```

Comment:
@TomFischer Would be nice if the Point3d could be directly initialized from an initializer list.